Makefile

```diff
@@ -17,20 +19,29 @@ check: check_tools get_vendor_deps
 ########################################
 ### Build
 
+# build and install build a binary inside docker
 build:
-	go build $(BUILD_FLAGS) -o build/tendermint ./cmd/tendermint/
+	@XC_OS='$(GOOS)' XC_ARCH='$(GOARCH)' BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/dist.sh'"
+	@cp ./build/pkg/$(GOOS)_$(GOARCH)/tendermint ./build
 
-build_race:
-	go build -race $(BUILD_FLAGS) -o build/tendermint ./cmd/tendermint
+install: build
+	@mkdir -p $(GOPATH)/bin
+	@cp ./build/tendermint $(GOPATH)/bin
+
+# dev-* targets build a binary locally
+dev-build:
```

Comment:
I would remove these, and reintroduce build-race.

Reply:
I can't comment on build_race, possibly the devs need it. As for the `install` target, I want to praise the method described here and point out that we should not use `go install`. `go install` rebuilds in most cases (which defeats the purpose of `go build` and adds extra time) and copies cached packages - which we don't have. Copying the binary, like here,  should be perfectly fine.